### PR TITLE
BTE-94: add group by formatted_date for getDailyAssetForAssetId

### PR DESCRIPTION
**Pull request by zhihuizhangjaden**
## Context

* Describe the changes made and why they were made.

Fix errors:
```... must appear in the GROUP BY clause or be used in an aggregate function```

Fixed endpoints:
https://api.parallel.fi/para/assets/v1/prices/snapshots/:assetId?unit=hourly
https://api.parallel.fi/heiko/assets/v1/prices/snapshots/:assetId?unit=hourly

* Provide links to any issues or designs for this change.

## Were any new external packages added?

* List external packages added and why they were added

### CODE REVIEW

File: packages/app/src/service/assetService.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const getDailyAssetForAssetId = async (
       .from(AssetEntity, 'asset_entity')
       .where('asset_id = :assetId', { assetId })
       .groupBy('asset_id')
```
Comment (crystalynliu) [inline packages/app/src/service/assetService.ts:50]: I think no need to group by asset_id, is it? Since there is a where selection for asset_id above.

Comment (zhihuizhangjaden) [reply packages/app/src/service/assetService.ts:50]: `asset_id` & `formatted_date` need to be used in groupBy, but `max_date` doesn't.

See this link https://stackoverflow.com/questions/13325583/postgresql-max-and-group-by

Comment (crystalynliu) [reply packages/app/src/service/assetService.ts:50]: I mean `where('asset_id = :assetId', { assetId })` is already filter by one assetId, why do we need group by assetId again?

Comment (zhihuizhangjaden) [reply packages/app/src/service/assetService.ts:50]: This is the inner join table, we must asset_id so then can do `on a.asset_id = b.asset_id and a.block_timestamp = b.max_date` operation

Comment (crystalynliu) [reply packages/app/src/service/assetService.ts:50]: Oh Got it.

**Review by crystalynliu — APPROVED** [review]

**Pull request merged**